go.mod

```diff
@@ -1,16 +1,15 @@
 module github.com/wowsims/wotlk
 
-go 1.18
+go 1.21
 
 require (
-	github.com/golang/protobuf v1.5.3
 	github.com/google/go-cmp v0.5.8
 	github.com/pkg/browser v0.0.0-20210911075715-681adbf594b8
 	github.com/satori/go.uuid v1.2.0
```

Comment:
should be replaced by github.com/google/uuid v1.3.1. the single use-case is in web/main.go:139, and becomes `newID := uuid.NewString()`.